bindings/rust/s2n-tls/src/client_hello.rs

```diff
@@ -222,115 +391,16 @@ impl fmt::Debug for ClientHello {
         let session_id = self.session_id().map_err(|_| fmt::Error)?;
         let session_id = hex::encode(session_id);
         let message_head = self.raw_message().map_err(|_| fmt::Error)?;
-        let mut hash = Vec::new();
-        self.fingerprint_hash(FingerprintType::JA3, &mut hash)
-            .map_err(|_| fmt::Error)?;
```

Comment:
Do we have to stop fingerprinting in the debug message? We don't have to make the unstable APIs available in order to include the fingerprint in our internal code.

Reply:
Or minimally, shouldn't we still put the fingerprint in the debug message if fingerprinting is enabled?

Reply:
I've spent a lot of time fiddling with this. I still don't think there is a way to do this. Even if I can get it working, I end up with clippy errors which isn't good.

Reply:
okay nvm, I think I just needed a break from this. This new commit should work.